### PR TITLE
Educator page mini update

### PR DESCRIPTION
**Pull request by picklesrus**
Adds link to teacher account setup guide on the educator page where you sign up for an educator account.
I will double check the link destination is the final one before committing. 
image

### CODE REVIEW

File: src/views/teachers/landing/landing.jsx [hunk 1]
```diff
@@ -338,7 +338,21 @@ const Landing = props => (
                         <FormattedMessage id="teacherlanding.accountsTitle" />
                     </h2>
                     <p>
-                        <FormattedHTMLMessage id="teacherlanding.accountsDescription" />
+                        <FormattedMessage
```
Comment (apple502j) [inline src/views/teachers/landing/landing.jsx:341]: That was the only place where FormattedHTMLMessage was used - so you can remove it

**Comment by apple502j** [comment]
also why Google Documents? We had problems with "blank 3.0 FAQ" in the past and are we doing it again?

File: src/views/teachers/landing/landing.jsx (hunk 1 above)
Comment (picklesrus) [reply src/views/teachers/landing/landing.jsx:340]: Thanks! That probably explains the lint error too!

**Comment by picklesrus** [comment]
Yes, we're experimenting with using a Google Doc instead here based on some feedback from folks expressing preference to this format for variety of reasons.

**Comment by benjiwheeler** [comment]
Tested, looks great, links work!

**Comment by benjiwheeler** [comment]
@apple502j Can you remind me what the problem was in the past? Was it that Google Docs is blocked in some places?

**Comment by apple502j** [comment]
@benjiwheeler It's mostly from the mainland China - they reported that 3faq is a blank page.

**Pull request merged**